tests/e2e/specs/myOrders_spec.js

```diff
@@ -54,6 +55,18 @@ describe('my orders', () => {
         cy.wrap($order)
           .find('[data-test=order-number]')
           .contains('1234');
+        cy.get('[data-test=location-selector-open-button]').click();
+        cy.get('span[data-test=location-selector-dropdown]')
+          .click()
+          .parent()
+          .contains('Deutsch')
+          .click();
```

Comment:
It breaks a bit the flow of the test changing the language now. I understand why you need another language, but you could change the language when starting the test, not in the middle.

Reply:
for some weird reason it doesn't find the location selector in the beginning of test 😕

Reply:
Then find the reason why, because that may be a bug.